libbeat/beat/beat.go

```diff
@@ -303,3 +317,20 @@ func handleError(err error) error {
 func GetDefaultVersion() string {
 	return defaultBeatVersion
 }
+
+func startCollectingMetrics() error {
```

Comment:
We should remove this and replace it with a method call every time the metrics are needed. So no need to define an interval which will lead to the problem, that metrics are not up-to-date when they are fetched.